AdobeMediaGalleryApi/Model/Keyword/Command/GetAssetKeywordsInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeMediaGalleryApi\Model\Keyword\Command;
+
+/**
+ * Interface GetAssetKeywordsInterface
+ * @api
+ */
+interface GetAssetKeywordsInterface
+{
+    /**
+     * Get asset related keywords.
+     *
+     * @param int $assetId
+     *
+     * @return \Magento\AdobeMediaGalleryApi\Api\Data\KeywordInterface[]
+     */
+    public function execute(int $assetId): array;
```

Comment:
I find it better to keep function parameter names generic (makes it easier to copy-paste))
```suggestion
    public function execute(int $id): array;
```